SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -259,7 +259,8 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 
 	if (AActor* EntityActor = EntityRegistry->GetActorFromEntityId(EntityId))
 	{
-		UE_LOG(LogSpatialReceiver, Log, TEXT("Entity for actor %s has been checked out on the worker which spawned it."), *EntityActor->GetName());
+		UE_LOG(LogSpatialReceiver, Log, TEXT("Entity for actor %s has been checked out on the worker which spawned it or is a singleton previously created on this worker"), \
```

Comment:
Or a singleton _linked_ on this worker (not necessarily created on this worker)

Reply:
I meant the singleton actor was previously manually created on this worker. But yes, I'll re-word.